game_eggs/steamcmd_servers/truck-simulator/euro-truck-simulator2/egg-euro-truck-simulator2-dedicated-server.json

```diff
@@ -99,6 +79,26 @@
             "user_editable": true,
             "rules": "required|string|max:64",
             "field_type": "text"
+        },
```

Comment:
This is not needed and should be removed

Reply:
User and password

Reply:
you mean the query port or the AUTH? as username and password are nuablle

Reply:
> you mean the query port or the AUTH? as username and password are nuablle

User and password

idk why its used the wrong line :/